src/App.jsx

```diff
@@ -1,35 +1,23 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useDispatch } from 'react-redux';
+import './App.css';
+import Greeting from './components/Greeting';
+import { fetchGreeting } from './redux/slices/greeting/greetingSlice';
 
 function App() {
-  const [count, setCount] = useState(0)
+  const dispatch = useDispatch();
+
+  const refreshGreeting = () => {
+    dispatch(fetchGreeting());
+  };
 
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
+    <div className="text-center text-red-800">
+      <h1 className="h-36 w-4/5 font-black">
+        <Greeting />
+      </h1>
+      <button type="button" onClick={refreshGreeting} className="mt-8">Refresh</button>
+    </div>
+  );
```

Comment:
As mentinoned earlier in accordance to the requirement, kindly ensure that your `App` component has a `route` that leads to the Greeting component. A typical example is illustrated below, you might use it as it is applicable to your work. ℹ️ 🙏 

```
import React from 'react';
****import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';****
import './App.css';
import Greeting from './components/Greeting';
import { fetchGreeting } from './redux/slices/greeting/greetingSlice';

// Define your App component
function App() {
  return (
    <Router>
      <div className="text-center text-red-800">
        <Switch>
          {/* Route for the Greeting component */}
          <Route path="/greeting" component={Greeting} />
          {/* You can add more routes here if needed */}
        </Switch>
      </div>
    </Router>
  );
}

export default App;
```